src/main/java/racingGame/model/Car.java

```diff
@@ -0,0 +1,24 @@
+package racingGame.model;
+
+public class Car {
+    
+    public static final int INIT_LOCATION = 1;
+    public static final int CAR_FORWARD_CRITERIA = 3;
+    private int location;
+    
+    public Car() {
+        this.location = INIT_LOCATION;
+    }
+    
+    public int forward() {
```

Comment:
```suggestion
    public void forward(randomNum) {
         if(isForwardByRandom(randomNum) {
             // 이동
         }
    }
```

forward와 isForwardByRandom을 분리하지 않고 위와 같이 하나로 통합해 구현하는 것은 어떨까?